app/views/posts/index.html.erb

```diff
@@ -1 +1,30 @@
 <h2>Here is a list of posts for a given user</h2>
+<div>
+    <img src="<%= @user.photo %>" />
+</div>
+<div>
+<p><%= @user.name %></p>
+<p>Number of Posts: <%= @user.posts_counter %></p>
+</div>
+<% @user.posts.each do |post| %>
+ <div>
+ <p><%= link_to user_post_path(@user, post.id) do %></p>
+ <p>Post <%= post.id  %></p>
+ <% end %>
+ <p><%= post.text  %></p>
+ <p>Comments <%= post.comments_counter  %></p>
```

Comment:
- [ ] `@post.most_recent_comments` is missing in your post _index_ view. As this activity requires that we use the `recent_comments` method defined in the _post_ model, in your case `most_recent_comments`, to render the user's 5 most recent comments in the `app/views/posts/index.html.erb` view, it should be similar to:

```ruby
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

Kindly ensure to improve upon the same.